src/claimtrie.h

```diff
@@ -28,6 +34,30 @@
 
 uint256 getValueHash(COutPoint outPoint, int nHeightOfLastTakeover);
 
+// lower-case and normalize any input string name
+// see: https://unicode.org/reports/tr15/#Norm_Forms
+inline const std::string normalizeClaimName(const std::string& name)
```

Comment:
Returning const on non reference is non-sense in C++

Reply:
I'm keeping inline in place of a storage specifier though, will remove the const.